### PR TITLE
feat: comment on i18n failure

### PR DESCRIPTION
**Pull request by naomi-lgbt**
Checklist:

- [X] I have read and followed the [contribution guidelines](https://contribute.freecodecamp.org).
- [X] I have read and followed the [how to open a pull request guide](https://contribute.freecodecamp.org/#/how-to-open-a-pull-request).
- [X] My pull request targets the `main` branch of freeCodeCamp.
- [X] I have tested these changes either locally on my machine, or GitPod.

Related #49698 

Okay, this has been [tested](https://github.com/naomi-lgbt/actions-testing/pull/63) and is ready to go.

The logic here is that if the localisation tests fail, but the English tests pass, that's a potential indicator that we need to do a sync (not always, tests can be flaky, but it'll let the contributor know to reach out to us).

### CODE REVIEW

File: .github/workflows/node.js-tests.yml [hunk 1]
```diff
@@ -197,3 +197,29 @@ jobs:
...
+            org: "freeCodeCamp",
+            team_slug: "dev-team",
+            username: context.payload.pull_request.user.login
+          }).catch(() => ({status: 404}));
+          if (context.payload.pull_request.user.login !== "camperbot" && isDev.status !== 200) {
+            github.rest.issues.createComment({
+              issue_number: context.issue.number,
+              owner: context.repo.owner,
+              repo: context.repo.repo,
+              body: "Hi,\n\nIt looks like the localization tests have failed. This is normal when there are significant curriculum changes in the English version.\n\nPlease check with the dev-team on [Discord]() or our [forums](https://forum.freecodecamp.org) if this means a Crowdin sync is required to address the scenario. They will confirm and take over helping you through the PR.\n\nThanks for your understanding and contribution."
```
Comment (ShaunSHamilton) [inline .github/workflows/node.js-tests.yml:221]:
```suggestion
              body: "Hi,\n\nIt looks like the localization tests have failed. This is normal when there are significant curriculum changes in the English version.\n\nPlease check with the dev-team on [Discord](https://discord.gg/KVUmVXA) or our [forums](https://forum.freecodecamp.org) if this means a Crowdin sync is required to address the scenario. They will confirm and take over helping you through the PR.\n\nThanks for your understanding and contribution."
```

**Review by ShaunSHamilton — APPROVED** [review]
I am sad dev-team members do not get a nice message 🙃

**Comment by naomi-lgbt** [comment]
> I am sad dev-team members do not get a nice message 🙃

That is borrowed from the no translation PR, because when we're intentionally breaking things we don't want to get spammed by the bot (:

File: .github/workflows/node.js-tests.yml [hunk 2]
```diff
@@ -197,3 +197,29 @@ jobs:
...
+            org: "freeCodeCamp",
+            team_slug: "dev-team",
+            username: context.payload.pull_request.user.login
+          }).catch(() => ({status: 404}));
+          if (context.payload.pull_request.user.login !== "camperbot" && isDev.status !== 200) {
+            github.rest.issues.createComment({
+              issue_number: context.issue.number,
+              owner: context.repo.owner,
+              repo: context.repo.repo,
+              body: "Hi,\n\nIt looks like the localization tests have failed. This is normal when there are significant curriculum changes in the English version.\n\nPlease check with the dev-team on [Discord](https://discord.gg/KVUmVXA) or our [forums](https://forum.freecodecamp.org) if this means a Crowdin sync is required to address the scenario. They will confirm and take over helping you through the PR.\n\nThanks for your understanding and contribution."
```
Comment (raisedadead) [inline .github/workflows/node.js-tests.yml:221]:
```suggestion
              body: "Hi,\n\nIt looks like the localization tests have failed. This is expected when there are significant curriculum changes in the English version.\n\nThis could mean a Crowdin sync is required to address the scenario. Please check with our staff developer team, either on [Discord](https://discord.gg/KVUmVXA) or our [forums](https://forum.freecodecamp.org). They will confirm and take over helping you through the PR.\n\nThanks for your understanding and contribution."
```

**Review by raisedadead — CHANGES_REQUESTED** [review]
Minor change to the verbiage to make it more succinct.

**Review by raisedadead — APPROVED** [review]

**Pull request merged**